docs: incorporate comments from team review (#200)
Open-Attestation/documentation-website: docs/ethereum-section/wrap-document-eth.md

  |-- certificate-2.json
```

> In the `wrapped-documents` directory, you will find the wrapped documents that will be issued on the document store using `merkleRoot` in the next article. 

Reviewer: ```suggestion
In the `wrapped-documents` directory, you will find the wrapped documents that will be issued to the document store in the next article. 
```

On a side note, I noticed that we are inconsistent in the phrasing for `issued on the document store` or `issued to the document store`, let's standardize this to the latter.

Author: After searching for `on document store` and `on the document store`, only one more article was found & updated along with this one.
